### PR TITLE
Chiel add automerge capabilities

### PR DESCRIPTION
**Pull request by cfernhout**

### CODE REVIEW

File: .github/workflows/automerge.yaml
Comment (gtoonstra) [inline .github/workflows/automerge.yaml:4]: Only recommended for testing?

Comment (cfernhout) [reply .github/workflows/automerge.yaml]: This better?

File: .github/workflows/automerge.yaml
Comment (gtoonstra) [inline .github/workflows/automerge.yaml:1]: @cfernhout : I just thought of something.

This is a public repo. Someone can in theory raise a PR, then have a friend "approve" it maybe?  And then it automerges?  Or they can't because they're all read-only (not in the team)?

We should check that approval is only with us and not just anybody, and it cannot be abused that way.

**Review by gtoonstra — APPROVED** [review]

File: .github/workflows/automerge.yaml
Comment (cfernhout) [reply .github/workflows/automerge.yaml:1]: First-time commiters need to have approval. If we want we could also implement code owners such that at least a review from you or me is required.

**Review by devops-datafold — APPROVED** [review]

**Pull request merged**
